gfxr_ext/decode/dive_annotation_processor.cpp

```diff
@@ -56,14 +56,38 @@ void DiveAnnotationProcessor::WriteBlockEnd(const gfxrecon::util::DiveFunctionDa
         {
             uint64_t cmd_handle = args["commandBuffer"];
 
-            // There could be cases where vkBeginCommandBuffer is called after vkEndCommandBuffer
-            // without having the command buffer submitted
-            if (vkCmd.name == "vkBeginCommandBuffer")
+            if (vkCmd.name.find("vkBeginCommandBuffer") != std::string::npos)
             {
                 m_cmd_vk_commands_cache[cmd_handle].clear();
+
+                // Total draw count for this command buffer
```

Comment:
nit: i feel this comment should be put where we declare `begin_command_buffer_draw_call_count` rather than here. Here is more like to clear the count because we have encountered cases where vkBeginCommandBuffer is called multiple times for the same cmd within a frame, so we need to clear the count here whenever vkBeginCommandBuffer is called.